src/stateless/multiselect.vue

```diff
@@ -70,6 +70,7 @@ export default {
         options: { type: Array, required: true },
         autoReorder: { type: Boolean, default: true },
         isSearchable: { type: Boolean, default: false },
+        showSelectClearAll: { type: Boolean, default: true },
```

Comment:
We need to control this in same way as `isSearchable` fro VCE project. Should be confirmed by product review anyway.